src/components/modal/modal.tsx

```diff
@@ -1,15 +1,17 @@
 import { ReactNode, useState } from "react";
 import { Dialog, Transition } from "@headlessui/react";
-import exitButton from "../../images/exit-modal-button-min.svg";
-import { ButtonIcon } from "../button/button";
+import { Button } from "../button/button";
+import { faX } from "@fortawesome/free-solid-svg-icons";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
```

Comment:
import { IconDefinition } from "@fortawesome/fontawesome-common-types";
Try switching to IconDefinition instead to match type and replacing the IconProp. I switched it and the error went away.

Reply:
I made the changes, and this did resolve the error. Thanks!